plans/feat-scheduler-phase3.md

```
@@ -0,0 +1,117 @@
+  missedRunPolicy: MissedRunPolicy;
+  enabled: boolean;
+  roleId: string;       // default: DEFAULT_ROLE_ID
+  prompt: string;       // message sent to startChat
+  createdAt: string;    // ISO UTC
+  updatedAt: string;
+}
+```
+
+## API Endpoints
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)